src/Controller/LottoController.js

```diff
@@ -0,0 +1,85 @@
+import View from '../View/View.js';
+import Lotto from '../Model/Lotto.js';
+import { Random } from '@woowacourse/mission-utils';
+import Constant from '../Constants/Constant.js';
+
+class LottoController {
+  #view;
+
+  constructor() {
+    this.#view = View;
+  }
+
+  async play() {
+    const purchaseAmount = await this.#view.getPurchaseAmount();
+    const countOfLotto = this.#getCountOfLotto(purchaseAmount);
+    this.#view.printCountOfLotto(countOfLotto);
+    const lottoArray = this.#getLottoNumber(countOfLotto);
+    this.#view.printLottoNumbers(lottoArray);
+    const goalLotto = await this.#getGoalLotto();
+    const bonusNumber = await this.#view.getBonusNumber(goalLotto.getNumbers());
+    const correctArray = goalLotto.calculateCorrectNumber(lottoArray, bonusNumber);
+    const resultArray = this.#calculateResult(correctArray);
+    const rateOfReturn = this.#calculateRateOfReturn(resultArray, purchaseAmount);
+    this.#view.printResult(resultArray, rateOfReturn * 100);
+  }
+
+  #getCountOfLotto(purchaseAmount) {
+    return purchaseAmount / Constant.UNIT_OF_AMOUNT;
+  }
+
+  #getLottoNumber(countOfLotto) {
+    const lottoArray = [];
+    for (let i = 0; i < countOfLotto; i++) {
+      const lotto = this.#getRandomNumber();
+      lottoArray.push(lotto);
+    }
```

Comment:
이건 개인 취향일 수도 있는데 이런 로직은 배열 메소드를 이용해보는 건 어떻게 생각하시나요? 🙂
```suggestion
    const lottos = new Array(countOfLotto).fill().map((_) => this.#getRandomNumber());
```
예를 들면 이런 식으로요!

depth가 깊어지지 않고 의미가 보다 직관적으로 느껴진다는 장점이 있는 것 같아서, 저는 배열 메소드를 자주 활용하는 것 같아요 ㅎㅎ